src/pages/Collections/CollectionsPage.svelte

```diff
@@ -22,13 +104,34 @@
     Collections List</Pane
   >
   <Pane size={$collapsibleState ? 100 : $collectionRightPanelWidth}>
-    Tabs
+    <TabBar
+      bind:tabList={$tabList}
+      onNewTabRequested={_collectionPageViewModel.createNewTab}
+      onTabClosed={closeTab}
+      onDropEvent={_collectionPageViewModel.onDropEvent}
+      onDragStart={_collectionPageViewModel.handleDropOnStart}
+      onDropOver={_collectionPageViewModel.handleDropOnEnd}
+      onTabSelected={_collectionPageViewModel.handleActiveTab}
+    />
     <br />
     <Route>
       <RestExplorer />
     </Route>
   </Pane>
 </Splitpanes>
 
+<CloseConfirmationPopup
+  isOpen={closePopup}
```

Comment:
rename the variable `closePopup` to be something more readable, eg. `isPopupClosed`

Reply:
Done